DropKit.cpp

```diff
@@ -9,7 +9,7 @@ DropKit::DropKit() {}
 
 void DropKit::setup() {
   dispenser.attach(42);
-  dispenser.write(90);   // Initialize the dispenser in 90 degrees(in the middle).
+  dispenser.write(kDispenserDegrees90);   // Initialize the dispenser in 90 degrees(in the middle).
```

Comment:
Nice, just add the comment on the previous line:
// Initialize the dispenser in 90 degrees(in the middle).
dispenser.write(kDispenserDegrees90);